app/styles/ui/_text-box.scss

```
@@ -24,6 +24,10 @@
     margin-bottom: var(--spacing-third);
   }
 
+  :invalid{
+    border-color: $red-500;
```

Can we use our CSS variable for errors here, that way individual themes can modify it.

```suggestion
    border-color: var(--error-color);
```